dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -737,7 +737,9 @@ def _DownloadResults(self,
 
     if not list_results:
       self.logger.info(f'No results for flow ID {str(flow)}')
-      return []
+      # Insert an empty dataframe to represent the flow completed without
+      # result.
+      return [pd.DataFrame()]
```

Comment:
Wouldn't returning an empty list make more sense here? Otherwise this means "there was a result, but no rows were returned"

Reply:
Returning empty list here and added logic to the caller (Process()) to hande when the result had no rows.